### ISSUE TITLE
WARNING: erroneous pipeline: no element "gvadetect"

### ISSUE THREAD

**Issue by ntackie** [dlstreamer/dlstreamer#233]
Hello 
 Please when i try to initialize the contents of gstreamer - dlstreamer elements  i get the error below. 

**ty@ty:~/intel/dlstreamer_gst/scripts$  source ~/intel/dlstreamer_gst/scripts/setup_env.sh
bash: source ~/intel/dlstreamer_gst/scripts/setup_env.sh: No such file or directory**

Also I proceeded to running a pipeline using the code i got the error below 

**ty@ty:~$ gst-launch-1.0 filesrc location=$/home/ty/Downloads/person-bicycle-car-detection.mp4 ! decodebin ! gvadetect model=$~/intel/dl_streamer/models/intel/person-vehicle-bike-detection-2004/FP32/person-vehicle-bike-detection-2004.xml model_proc=${DETECTION_MODEL_PROC} device=CPU ! queue ! gvawatermark ! videoconvert ! fpsdisplaysink video-sink=xvimagesink sync=false
WARNING: erroneous pipeline: no element "gvadetect"**

Can you please provide assistance i am running gstreamer from a Ubuntu VM

**Comment by brmarkus** [comment]
Can you describe your environment in more details, first, please?
You have an Ubuntu VM? Which version of Ubuntu?
Have you installed OpenVINO, which version, how? Built from source-code or installed a pre-built version?
How have you installed DL-Streamer, as part of an OpenVINO installation or separately, from source code?

Can you describe your installation procedure, please?

If you install OpenVINO as described under "https://docs.openvinotoolkit.org/latest/openvino_docs_install_guides_installing_openvino_linux.html", then make sure you enable the installation of DL-Streamer.
Ideally you have an Intel GPU and Intel CPU available as the underlying HW for your Ubuntu VM.

Where did you get the command "source ~/intel/dlstreamer_gst/scripts/setup_env.sh" from? From the error message it looks like you don't have that file, you don't have that folder structure?

**Comment by ntackie** [comment]
hi @brmarkus 

I am using Ubuntu 20.04 on Oracle VM. I have OpenVINO version 2021.4.689 and i installed using the Intel Distribution Toolkit with this doc as my guide - https://docs.openvinotoolkit.org/latest/openvino_docs_install_guides_installing_openvino_linux.html. I also installed using DL-streamer using the installation guide from this github. 

I got this command source ~/intel/dlstreamer_gst/scripts/setup_env.sh when i was going through the tutorial example to check from this github rep. The command is suppose to initialize/set the environment.
